package on2015_03.on2015_03_03_Codeforces_Round__293__Div__2_.E___Arthur_and_Questions;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2015_03/on2015_03_03_Codeforces_Round__293__Div__2_/E___Arthur_and_Questions/E - Arthur and Questions.task"))
			Assert.fail();
	}
}
